<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectExplorer::Internal::MakeStep</class>
 <widget class="QWidget" name="ProjectExplorer::Internal::MakeStep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>279</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="makeLabel">
     <property name="text">
      <string>Override %1:</string>
     </property>
     <property name="buddy">
      <cstring>makeLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Utils::PathChooser" name="makeLineEdit" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="makeArgumentsLabel">
     <property name="text">
      <string>Make arguments:</string>
     </property>
     <property name="buddy">
      <cstring>makeArgumentsLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="makeArgumentsLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="jobsLabel">
     <property name="text">
      <string>Parallel jobs:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="targetsLabel">
     <property name="text">
      <string>Targets:</string>
     </property>
     <property name="buddy">
      <cstring>targetsList</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QListWidget" name="targetsList"/>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="jobLayout">
     <item>
      <widget class="QSpinBox" name="userJobCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="overrideMakeflags">
       <property name="text">
        <string>Override MAKEFLAGS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nonOverrideWarning">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
